support/commands.js

```diff
@@ -443,4 +443,12 @@ Cypress.Commands.add('switchToExtensionWindow', () => {
 
 Cypress.Commands.add('switchToExtensionRegistrationWindow', () => {
   return cy.task('switchToExtensionRegistrationWindow');
-});
\ No newline at end of file
+});
+
+Cypress.Commands.add('makeOfferAndConfirmTransaction', () => {
+  cy.contains('agoric1p2aqakv3ulz4qfy2nut86j9gx0dx0yw09h96md');
+  cy.contains('Make an Offer').click();
+  return cy.confirmTransaction().then(taskCompleted => {
+    expect(taskCompleted).to.be.true;
+  });
+});
```

Comment:
these functions should not contain logic related to our offer up dapp since these will be part of the synpress framework and will be exposed to all users